doc/spec/api.md

```diff
@@ -608,16 +606,18 @@ image manifest. An image can be pushed using the following request format:
        ...
     }
 
+**TODO**: do you mean `tag` field in the next sentence (or is the JSON out of date)?
```

Comment:
in the JSON, there's no `reference` field, so there's something incongruous here.